dune/grid/common/WellConnections.cpp

```diff
@@ -31,27 +31,25 @@ namespace Dune
 {
 namespace cpgrid
 {
-WellConnections::WellConnections(const OpmEclipseStateType& eclipseState,
+WellConnections::WellConnections(const std::vector<const OpmWellType*>& wells,
                                  const std::array<int, 3>& cartesianSize,
                                  const std::vector<int>& cartesian_to_compressed)
 {
-    init(eclipseState, cartesianSize, cartesian_to_compressed);
+    init(wells, cartesianSize, cartesian_to_compressed);
 }
 
-void WellConnections::init(const OpmEclipseStateType& eclipseState,
+void WellConnections::init(const std::vector<const OpmWellType*>& wells,
                            const std::array<int, 3>& cartesianSize,
                            const std::vector<int>& cartesian_to_compressed)
 {
 #if HAVE_OPM_PARSER
-    std::vector<const Opm::Well*>  wells  = eclipseState.getSchedule().getWells();
-    int last_time_step = eclipseState.getSchedule().getTimeMap().size()-1;
     well_indices_.resize(wells.size());
 
     // We assume that we know all the wells.
     int index=0;
     for (const auto well : wells) {
         std::set<int>& well_indices = well_indices_[index];
-        const auto& completionSet = well->getCompletions(last_time_step);
+        const auto& completionSet = well->getCompletions( );
```

Comment:
Is this equivalent? I assume that no arguments would give all completions over the whole history of the well, and last_time_step only those present at the end? If so, then this would actually be a bugfix, since if a well loses some completions before the end they should still be considered for the partitioning!

Reply:
They are 100% equivalent. Completions can not disappear.

Reply:
Thanks for the clarification! I guess I was a little confused between completions appearing/disappearing, and being opened/closed. I'll restart the Jenkins test just to see if there was a temporary hiccup.